builder.conf

```diff
@@ -2,3 +2,9 @@
 SERVER_STATE_DIR = /var/lib/update
 BUNDLE_DIR = /home/clr/mix/bundles
 YUM_CONF = /usr/share/defaults/bundle-chroot-builder/yum.conf
+
+[URL]
```

Comment:
Since these options are swupd-specific, perhaps the section name could be [SWUPD] ?